src/AmqpBundle/DependencyInjection/Configuration.php

```diff
@@ -80,31 +83,6 @@ protected function addProducers(ArrayNodeDefinition $node)
 
                                     // args
                                     ->arrayNode('arguments')
-                                    ->prototype('scalar')->end()
-                                    ->defaultValue(array())
-                                    ->normalizeKeys(false)
-                                    ->end()
-
-                                    // binding
-                                    ->arrayNode('routing_keys')
```

Comment:
Why have you removed the routing keys node here ?
this node is used here : https://github.com/M6Web/AmqpBundle/pull/25/files#diff-030150e6243abd1d512cc37c8c359dd8R105

But maybe i've missed something !